src/assets/src/components/queueSettings.tsx

```diff
@@ -6,308 +6,432 @@ import { faChevronLeft } from "@fortawesome/free-solid-svg-icons";
 import { Button, Col, Nav, Row, Tab } from "react-bootstrap";
 
 import {
```

Comment:
Are those all formatting changes, not related to the page title change? If so, it would be good to keep this PR clean and focused.